src/components/AlertDetail.vue

```diff
@@ -450,7 +458,6 @@
                         <span class="label">
                           {{ item.status | capitalize }}
                         </span>
-                      </div>
```

Comment:
Please re-add this `<div>`

Reply:
Oh, sorry, my mistake. I re-added this line.